src/xcode/ENA/ENA/Source/Services/CCLService/__tests__/DCCWalletInfo+Fake.swift

```diff
@@ -16,7 +16,8 @@ extension DCCWalletInfo {
 		mostRelevantCertificate: DCCCertificateContainer = .fake(),
 		verification: DCCVerification = .fake(),
 		validUntil: Date = Date(),
-		certificateReissuance: DCCCertificateReissuance? = nil
+		certificateReissuance: DCCCertificateReissuance? = nil,
+		certificatesRevokedByInvalidationRules: [DCCCertificateContainer] = [.fake()]
```

Comment:
```suggestion
		certificatesRevokedByInvalidationRules: [DCCCertificateContainer] = []
```
For fakes the default should usually be empty. The caller should not assume anything about the default parameters and set them if they are required for the test case.